static/elements/chromedash-feature-table.js

```diff
@@ -38,11 +48,25 @@ class ChromedashFeatureTable extends LitElement {
     `];
   }
 
+  renderMessages() {
```

Comment:
How is this function being notified if `this.loading` became `False`? In ploymer you can achieve it by creating a simple observers, eg. `loading` to the parameter

Reply:
Good catch.  This was a minor defect.

In lit-element, the render method is called after anything modified one of the element properties as defined that are the properties() method.  I had this.features listed as a property, but not this.loading.  The code worked because both this.features and this.loading were set at the same time.  However, you are correct that this.loading should also be a property.  That would make the code do the right thing if we somehow set this.loading at some other time, or if the render() method was called between those two assignments (but I don't think that that is possible in JS).

Reply:
I see. That seems like a much simpler design than creating many observable changes. Thanks for the explanation :)